include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -70,6 +70,10 @@ static const char* const kOrtSessionOptionsUseDeviceAllocatorForInitializers = "
 // "1": default, thread will spin a number of times before blocking
 static const char* const kOrtSessionOptionsConfigAllowInterOpSpinning = "session.inter_op.allow_spinning";
 static const char* const kOrtSessionOptionsConfigAllowIntraOpSpinning = "session.intra_op.allow_spinning";
+// Decide to Create default thread pool or not, if not, the thread pool size is always 1
+// The scenario is for some EPs which has its own threadpool, we don't want two more threadpool coexist
+// XNNPACK is one of the EPs, pthreadpool is used in XNNPACK
+static const char* const kOrtSessionOptionsConfigDisableThreadPool = "session.intra_op.disable_thread_pool";
```

Comment:
I don't think this config is required. Setting intro_op_num_threads to 1 should suffice as ORTs threadpool uses the calling thread as one of the threads in its pool.

Reply:
Indeed, what this config doing to is set intro_op_num_threads to 1 in `inference_sesstion.cc`.
But without this config, it's hard to prevent ort-devs/users setting intro_op_num_threads  to other numbers.

The motivation behind is to let dev/users config ort-threadpoo/pthreadpool more easily.